packages/amplify-e2e-core/src/categories/api.ts

```diff
@@ -609,19 +617,20 @@ export function updateRestApi(cwd: string, settings: Partial<typeof updateRestAp
     default:
       throw new Error(`updateOperation ${completeSettings.updateOperation} is not implemented`);
   }
-  chain.wait('Restrict API access').sendNo().wait('Do you want to add another path').sendNo().wait('Successfully updated resource');
+  chain.wait('Restrict API access').sendNo().wait('Do you want to add another path').sendNo()
+    .wait('Successfully updated resource');
   return chain.runAsync();
 }
 
 const allAuthTypes = ['API key', 'Amazon Cognito User Pool', 'IAM', 'OpenID Connect'];
 
-export function addApi(projectDir: string, settings?: any) {
+export function addApi(projectDir: string, settings?: any, requireAuthSetup = true) {
```

Comment:
the only functional changes in this file are in this function. Everything else is lint fixes